lib/modules/manager/sveltos/extract.spec.ts

```diff
@@ -0,0 +1,227 @@
+import { Fixtures } from '../../../../test/fixtures';
+import { extractPackageFile } from '.';
+
+const validProfile = Fixtures.get('validProfile.yml');
+const validClusterProfile = Fixtures.get('validClusterProfile.yml');
+const validEventTrigger = Fixtures.get('validEventTrigger.yml');
+const malformedProfiles = Fixtures.get('malformedProfiles.yml');
+const randomManifest = Fixtures.get('randomManifest.yml');
+
+describe('modules/manager/sveltos/extract', () => {
+  describe('extractPackageFile()', () => {
+    it('returns null for empty', () => {
+      expect(extractPackageFile('nothing here', 'sveltos.yml')).toBeNull();
+    });
+
+    it('returns null for invalid', () => {
+      expect(
+        extractPackageFile(`${malformedProfiles}\n123`, 'sveltos.yml'),
+      ).toBeNull();
+    });
+
+    it('return null for kubernetes manifest', () => {
+      const result = extractPackageFile(randomManifest, 'sveltos.yml');
+      expect(result).toBeNull();
+    });
+
+    it('return null if deps array would be empty', () => {
+      const result = extractPackageFile(
+        malformedProfiles,
+        'applications.yml',
+      );
+      expect(result).toBeNull();
+    });
+
+    it('return result for double quoted argoproj.io apiVersion reference', () => {
+      const result = extractPackageFile(
+        `
```

Comment:
```suggestion
        codeblock`
```
Use `codeblock` to allow indentation of the inline fixtures. 
https://github.com/renovatebot/renovate/blob/057049b134f7987e5029f435f3d2a797bdccd04f/lib/util/toml.spec.ts#L6